web/src/stream_popover.js

```diff
@@ -604,6 +610,22 @@ export function register_stream_handlers() {
 }
 
 export function register_topic_handlers() {
+    // Toggle topic visibility_policy (currently in dev environment)
```

Comment:
Remove the part about "currently in dev ..." in the comment here. Writing comments like that is a bad idea because they almost always end up stale, since it's not next to the code that only enables this feature in the development environment, so when we do enable it for everyone, we'll very likely fail to update it here.